src/main/java/uk/gov/pay/api/validation/URLValidator.java

```diff
@@ -6,7 +6,7 @@
 
 public enum URLValidator {
     SECURITY_ENABLED {
-        private final UrlValidator URL_VALIDATOR = new UrlValidator(new String[]{"https"});
+        private final UrlValidator URL_VALIDATOR = new UrlValidator(new String[]{"https"}, ALLOW_LOCAL_URLS);
```

Comment:
Yeah, it's a bit controversial, but people will be coming from localhost for sandbox testing. Maybe we need some more discussion. I'll withdraw this change for now.